### PR TITLE
feat: Improve error messages

### PR DESCRIPTION
**Pull request by w3st3ry**
Related to #42 

Signed-off-by: Valentin Pichard <7628998+w3st3ry@users.noreply.github.com>

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report test#225.0 ✘  
* Stage 1
  * Test ✘

**Comment by geoffreybauduin** [comment]
Just a suggestion but it would be nice for your APIs to use the `%w` flag inside `fmt.Errorf` that has been introduced with go 1.13. See https://blog.golang.org/go1.13-errors

**Comment by ovh-cds** [comment]
CDS Report test#330.0 ✘  
* Stage 1
  * Test ✘

**Comment by w3st3ry** [comment]
@geoffreybauduin at this team we still compiling with go1.12. We gonna do this change in the future.

File: pkg/plugins/builtin/email/email.go [hunk 1]
```diff
@@ -48,8 +48,12 @@ func validConfig(config interface{}) error {
 		return errors.New("smtp_password is missing")
 	}
 
+	if cfg.SMTPPort == "" {
+		return errors.New("smtp_password is missing")
```
Comment (rbeuque74) [inline pkg/plugins/builtin/email/email.go]: Invalid error message

**Pull request merged**